### PR TITLE
Create upp-2.md

### PR DESCRIPTION
**Pull request by mrice32**
UPP 2 proposes an increase of the final fees for WETH and DAI.

### CODE REVIEW

File: UPPs/upp-2.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+It is extremely expensive, in terms of gas, for the DVM voters to vote on a price request. The DVM also pays out a relatively large number of rewards on each vote.
+To ensure that price requests remain rare, they need to be more expensive to discourage any sort of purposeful triggering.
+
+While this does make the capital requirements for liquidators and disputers higher, the increased size of the final fee lockup is unlikely to deter honest participants.
+
+## Specifics
+
+The WETH final fee will be increased to 0.5 WETH.
+The DAI final fee will be increased to 200 DAI.
```
Comment (chrismaree) [inline UPPs/upp-2.md:19]: Given the recent price increase in $UMA, I think these should be at least 2x this. I propose a target of 400USD so 1 WETH and 400 DAI.

Also, this specification does not mention renBTC which was added in UMIP-11

Comment (mrice32) [reply UPPs/upp-2.md:19]: Good point, I'll add it.

Comment (mrice32) [reply UPPs/upp-2.md:19]: @hlambur @nicholaspai thoughts?

Comment (hlambur) [reply UPPs/upp-2.md:19]: The increase makes sense to me.

Comment (mrice32) [reply UPPs/upp-2.md]: @chrismaree I have updated this to reflect a $400 final fee. PTAL.

**Review by chrismaree — APPROVED** [review]

**Pull request merged**